src/conductor.ts

```diff
@@ -101,6 +101,20 @@ export class Conductor {
     })
   }
 
+  connectSignals = async () => {
+    const url = this.testInterfaceUrl()
+    const { onSignal } = await this.webClientConnect({url})
+
+    onSignal((msg: {signal, instance_id: string}) => {
+      console.log(this.instanceMap)
+      const instances = Object.keys(this.instanceMap).map(key => this.instanceMap[key])
+      const instance = instances.find(instance => instance.id == msg.instance_id)
+      if(instance != undefined) {
```

Comment:
I think this is equivalent and clearer (`!==` would be a different story)

```suggestion
      if(!instance) {
```